documentation/en/source/pages/templates-spec/builders/builders-pxe.rst

```diff
@@ -0,0 +1,74 @@
+.. Copyright (c) 2007-2016 UShareSoft, All rights reserved
```

Comment:
I agree with @jgweir comment, please change the copyright here to 2017, and you could open an issue for old files